libsyclinterface/source/dpctl_sycl_queue_interface.cpp

```diff
@@ -38,6 +38,100 @@
+        {                                                                      \
+            auto la = local_accessor<char, NDIM>(R, CGH);                      \
+            CGH.set_arg(IDX, la);                                              \
+            return true;                                                       \
+        }                                                                      \
+        case DPCTL_SIGNED_CHAR:                                                \
+        {                                                                      \
+            auto la = local_accessor<signed char, NDIM>(R, CGH);               \
+            CGH.set_arg(IDX, la);                                              \
+            return true;                                                       \
+        }                                                                      \
+        case DPCTL_UNSIGNED_CHAR:                                              \
+        {                                                                      \
+            auto la = local_accessor<unsigned char, NDIM>(R, CGH);             \
+            CGH.set_arg(IDX, la);                                              \
+            return true;                                                       \
+        }                                                                      \
+        case DPCTL_SHORT:                                                      \
+        {                                                                      \
+            auto la = local_accessor<short, NDIM>(R, CGH);                     \
+            CGH.set_arg(IDX, la);                                              \
+            return true;                                                       \
+        }                                                                      \
+        case DPCTL_INT:                                                        \
+        {                                                                      \
+            auto la = local_accessor<int, NDIM>(R, CGH);                       \
+            CGH.set_arg(IDX, la);                                              \
+            return true;                                                       \
+        }                                                                      \
+        case DPCTL_UNSIGNED_INT:                                               \
+        {                                                                      \
+            auto la = local_accessor<unsigned int, NDIM>(R, CGH);              \
+            CGH.set_arg(IDX, la);                                              \
+            return true;                                                       \
+        }                                                                      \
+        case DPCTL_UNSIGNED_INT8:                                              \
```

Comment:
Is this different from `DPCTL_UNSIGNED_CHAR`? If these are aliases, we could never get good coverage

Reply:
Are you running coverage on `libsyclinterface`? https://coveralls.io/github/IntelPython/dpctl seems to indicate that `libsyclinterface` is not part of dpctl's coverage numbers. Moreover not all files of `libsyclinterface` are included.

Reply:
It is: 

![image](https://github.com/IntelPython/dpctl/assets/21087696/9ba6bd90-b266-46a1-955a-7f427cd29822)

Reply:
I see that many files from `libsyclinterface` are indeed absent from the report. I may have an idea why. I will open a PR to find out

Reply:
Well the aggregate may not be getting impacted if the overall lines of code for dpctl repo is not swayed much by libsycinterface. Even then it seems spurious.